### PR TITLE
Improve sizing and turn scaling on/off

### PR DESCRIPTION
**Pull request by Jephuff**
This adds `leftTextWidth` and `scaleToFit` to params. It also changes the behaviour of margin. Before merging I will update the readme/api

### CODE REVIEW

File: src/Histogram.js [hunk 1]
```diff
@@ -22,6 +22,11 @@ var OncoHistogram;
 OncoHistogram = function (params, s, rotated) {
     var _self = this;
 
+    _self.lineWidthOffset = 10;
+    _self.lineHeightOffset = 5;
```
Comment (andricDu) [inline src/Histogram.js]: can these be overridden by configuration?

Comment (Jephuff) [reply src/Histogram.js]: sure! they were just hardcoded before. What would they be called? `histogramBorderPadding`?

Comment (andricDu) [reply src/Histogram.js]: Yea that sounds great. My bad for not having them configurable before.

**Comment by andricDu** [comment]
ready to merge?

**Comment by Jephuff** [comment]
I'm just going to update the readme with the new params and then it's good to go.

**Comment by Jephuff** [comment]
updated api.md, ready to merge @andricDu

**Pull request merged**